src/commandDetails/games/blackjack.ts

```diff
@@ -0,0 +1,295 @@
+// Sapphire Specific:
+// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
```

Comment:
I think this is not needed for slashified commands. This is only for those commands that just run on Sapphire.
So you can remove the two line comments.

Reply:
Okie, I removed them.